source/operations/checklists/software.rst

```diff
@@ -8,4 +8,62 @@ Software Checklist
    :local:
    :depth: 2
 
-Stub - detail software checklist items
+Use the following checklist when planning the software configuration for a production, distributed MinIO deployment.
+
+Pre-MinIO Install Requirements
+------------------------------
+
+.. list-table::
+   :widths: auto
+   :width: 100%
+
+   * - :octicon:`circle`
+     - Servers running a Linux operating system, such as a supported version of Red Hat Enterprise Linux (RHEL) or Ubuntu LTS
```

Comment:
The key is that we assume Linux for Production environments.

More recent kernels (5.x) are preferred wherever possible. Right now that's RHEL9 and Ubuntu 18+.

Reply:
Clarified